src/axom/slam/tests/slam_map_BivariateMap.cpp

```diff
@@ -375,7 +375,7 @@ void constructAndTestBivariateMapIterator(int stride)
   {
     int idx2 = 0;
     int compIdx = 0;
-    auto begin_iter = m.begin(idx1);
+    //auto begin_iter = m.begin(idx1);

```

Comment:
@rhornung67 That's a good question.  I was just following a pattern in the same code.  Not sure what the reason are for commenting out instead of removing code in other places.  I'll see if anyone can explain, and if not, I'll clean up the obsolete code.